data/TeamData.js

```diff
@@ -817,64 +829,88 @@ const MEMBERS = [
           ],
         ],
       },
-      // {
-      //   sectionName: "First Year Representatives",
-      //   members: [
-      //     [
-      //       {
-      //         image: Name_Image,
-      //         description: ``,
-      //         name: "First Last",
-      //         role: "",
-      //       },
-      //       {
-      //         image: Name_Image,
-      //         description: ``,
-      //         name: "First Last",
-      //         role: "",
-      //       },
-      //     ],
-      //   ],
-      // },
+      {
+        sectionName: "First Year Representatives",
+        members: [
+          [
+            //Get Tony's Pic
+            {
+              image: Tony_Image,
+              description: ``,
+              name: "Tony Wang",
+              role: "",
+            },
+            // {
+            //   image: Mariam_Image,
+            //   description: ``,
+            //   name: "First Last",
+            //   role: "",
+            // },
+          ],
+        ],
+      },
+    ],
+    subcoms: [
+      {
+        icons: [faClipboard],
+        name: "HR",
+        description: `The HR portfolio is committed to strengthening teamwork and cross-portfolio communication within the society. They are responsible for organizing internal events, to help shape our culture at Co-op Soc.`,
+        members: ["Stella Lin", "James Liao", "Aiden Ahmad", "Rashid Abuzarov"],
```

Comment:
Formatting - one per line. Make sure to set up eslint / formatter, ask @lhvy for help if needed

Reply:
Don't think this was done, I meant something like:
```js
members: [
  "Stella Lin",
  "James Liao",
  "Aiden Ahmad",
  "Rashid Abuzarov",
]
```

Should be automated by the linter/formatter. Let me know if the formatter chose to do it this way, but looks more like you just not setting it up yet.